mongoDB-case-study.md

```diff
@@ -0,0 +1,113 @@
+---
+title: MongoDB Product Case Study
+keywords: development platforms, MongoDB project, NoSQL
+tags: Development Platforms, MongoDB Project, NoSQL
+sidebar: development-platforms
+permalink: development-platforms/mongoDB.html
+folder: development-platforms
+---
+
+## Introduction to MongoDB
+
+MongoDB is a open source NoSQL database management program that has focus on being scalable and flexible, allowing for storing of big amounts of data in a fast and agile database.
+MongoDB was created by the owners of a internet advertisement company, called DoubleClick, they struggled with the storage databases not being scalable and agile enough.
+Instead of using tables and rows, like the SQL databases, MongoDB gathers information and data in collections and documents.
+
+## Brief History
+
+In the early days, 1970s, NoSQL was just a supplement to the SQL ways of storing data. But as the internet grew and evolved, so did the users, the records and the size of data being stored. The SQL had limitations, and they became more and more apparent. So adopting a more flexible and scalable way to store data was a priority. This formed the evolution of NoSQL, and from that MongoDB arised.
+
+In the points below, I will list the startup points to MongoDB, but also the biggest releases added to the database throughout the years.
```

Comment:
In this style of writing we use:
- We `now we can look at this feature...`
- You `you can check this for yourself by visiting...`
- The author `In the opinion of the author, this is not enough...`

But we do not use:
- I

I is only used in opinion writing, whereas this is a technical document.